Config.example.toml

```diff
@@ -3,6 +3,9 @@
 # A discord token for the bot to login
 discord_token = 'Your token'
 
+# An API key for the Tenor API for commands like /cat, comment it out if you do not want such commands
+tenor_token = 'Your Tenor API key'
```

Comment:
Just a feedback, not a change request (feel free to resolve this once you read it)
`#[serde(alias=...)]` adds an alias, so this change wasn't necessary, as both `tenor_token` and `tenor_api_key` would work. I do however prefer `tenor_token`, so I'm glad you changed it.